package.json

```diff
@@ -11,12 +11,12 @@
     "node": ">=6"
   },
   "scripts": {
-    "install-all": "yarn && yarn install-launcher && yarn install-cli && yarn install-extension && yarn install-viewer",
+    "install-all": "yarn && concurrently --kill-others-on-fail \"yarn install-launcher\" \"yarn install-cli\" \"yarn install-extension\" \"yarn install-viewer\"",
```

Comment:
this [may or may not work on linux](https://github.com/kimmobrunfeldt/concurrently/issues/4#issuecomment-217124689), apparently